requirements_dev.txt

```diff
@@ -1,5 +1,6 @@
 coverage==6.3.2
-flake8==4.0.1
+Flake8-pyproject==1.2.3
+flake8==6.1.0
```

Comment:
had to upgrade flake8 so that it is compatible with the `flake8-pyproject` package

Reply:
Fair enough, it was quite old :)